src/docfx/lib/log/Telemetry.cs

```diff
@@ -65,26 +75,25 @@ public static void TrackCacheMissCount(TelemetryName name)
             s_cacheCountMetric.TrackValue(1, name.ToString(), "miss", s_os, s_version, s_repo, s_branch, s_correlationId);
         }
 
-        public static void TrackBuildItemCount(ContentType contentType)
+        public static void TrackBuildFileTypeCount(Document file)
         {
-            s_buildItemCountMetric.TrackValue(1, $"{TelemetryName.BuildItems}-{contentType}", s_os, s_version, s_repo, s_branch, s_correlationId);
+            var fileExtension = (Path.GetExtension(file.FilePath.Path) ?? "<null>").ToLowerInvariant();
```

Comment:
```suggestion
            var fileExtension = file.FilePath.Format.ToString();
```

Reply:
the format just support `json`, `md` and `yml`, what about resource? like `.jpg`

Reply:
yep you are right.
```suggestion
            var fileExtension = Path.GetExtension(file.FilePath.Path)?.ToLowerInvariant() ?? "<null>";
```